src/configs/base_report_templates.c4m

```diff
@@ -1662,6 +1663,8 @@ running insert commands.
   key._VIRTUAL.use                            = true
   key._CURRENT_HASH.use                       = true
   key._IMAGE_ID.use                           = true
+  key._REPO_TAGS.use                          = true
```

Comment:
Not sure these should be in the audit report? What was your logic there?

Reply:
I think github shows diff badly here. this is actually inside `report_template terminal_insert {`